components/nativewindui/DatePicker/DatePicker.android.tsx

```diff
@@ -22,37 +30,39 @@ export function DatePicker(
   return (
     <View className="flex-row gap-2.5">
       {props.mode.includes('date') && (
-        <View className="relative pt-1.5">
+        <View className={cn('relative pt-1.5', props.materialDateClassName)}>
           <Pressable
             onPress={show('date')}
-            className="rounded-md border border-border px-5 py-3 active:opacity-80">
-            <Text variant="subhead">
+            className="border-foreground/30 rounded border py-3 pl-2.5 active:opacity-80">
+            <Text className="py-px">
```

Comment:
@mrzachnugent can you elaborate on this change?

Reply:
@danstepanov When comparing the date-picker to the text-field from the before image, you can see why I changed:
- rounded -> the date-picker was more rounded than the text-field
- pl-2.5 -> the date-picker text was more to the right than the text-field's
- border-foreground/30 -> the date-picker's border was lighter than the text-fields
- py-px -> the date-picker was not as tall as the text-field